ragna/ui/js.py

```diff
@@ -0,0 +1,107 @@
+from .styles import MODAL_MAX_HEIGHT
+
+# Reset modal size
+MODAL_HACK = f"""
```

Comment:
Can we have them in native files, i.e. not `.py`? Like this, it is impossible to apply any formatting.

Reply:
Yes, open an issue for that. I won't move them all to separate files though, some are small and not worth it.